docs/contributor-guide/developer-guide/deploy-system/trouble-shooting.md

```diff
@@ -0,0 +1,27 @@
+# Deploy system Troubleshooting
+
+For Azure specific steps also see [Dev Azure](dev-azure.md) page.
+
+## I can't see my changes in the deployment
```

Comment:
If we don't have any text here, I think we should remove the heading

Reply:
I added some text